server.js

```diff
@@ -1,36 +1,282 @@
-import express from 'express'
-import bodyParser from 'body-parser'
-import cors from 'cors'
-import mongoose from 'mongoose'
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo"
+import express from "express";
+import bodyParser from "body-parser";
+import cors from "cors";
+import mongoose from "mongoose";
+
+import spotifyData from "./data/spotify-releases.json";
+import spotifyTypes from "./data/spotify-types.json";
+import spotifyArtists from "./data/spotify-artists.json";
+
+const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-spotify"
```

Comment:
great that you remember to change the local host to a matching name of the actual project